main.py

```diff
@@ -0,0 +1,103 @@
+""" SHEST FASTAPI SERVER """
+
+from cryptography.fernet import Fernet
+from fastapi import FastAPI, Request
+from os import popen
+from uvicorn import run
+from fastapi.responses import StreamingResponse
+
+
+app = FastAPI()
+
+
+def load_key():
+    """Load the previously generated key"""
+
+    with open('secret.key', 'r') as file:
+        return file.read()
+
+
+def encrypt_message(message: str):
+    """
+    Encrypts a message"
+    :param message: A regular message.
+    :return: The encrypted message.
+    """
+
+    key = load_key()
+    f = Fernet(key)
+    message = message.encode()
+    encrypted_message = f.encrypt(message)
+    return encrypted_message
+
+
+def decrypt_message(encrypted_message):
+    """
+    Decrypts an encrypted message
+    :param encrypted_message: The encrypted message to decrypt.
+    :return: The decrypted message.
+    """
+
+    key = load_key()
+    f = Fernet(key)
+    decrypted_message = f.decrypt(encrypted_message)
+    return decrypted_message
+
+
+async def stream_back(file_name: str):
```

Comment:
Change the function name to something more indicative. Not `stream_back`. I suggest `read_file`.